Exercise2/day2.sql

```diff
@@ -0,0 +1,117 @@
+-- Question 1 
+-- Write a SQL query to remove the details of an employee whose first name ends in ‘even’
+delete from employees where first_name like '%even';
+select * from employees where first_name like '%even';
+-- Question 2
+-- Write a query in SQL to show the three minimum values of the salary from the table.
+select * from employees order by salary limit 3;
+-- Question 3
+-- Write a SQL query to copy the details of this table into a new table with table name as Employee table and to delete the records in employees table.
```

Comment:
Q3 : Explore on implementing the same in a better way (without even defining the schema)